doc/index.md

```diff
@@ -21,11 +21,11 @@ width: 70%
 ---
 ```
 
----
+**`.hvplot()` is a powerful and interactive Pandas `.plot()` API**
```

Comment:
This sounds misleading :/ As `.hvplot` isn't just a Pandas `.plot` API but has a **much** larger scope.

Reply:
True

Reply:
I've added the word `like`. I agree it could be more precise, but don't have the wording. Do you have a suggestion?